packages/blockchain-wallet-v4-frontend/src/scenes/Earn/Learn/Learn.model.tsx

```diff
@@ -64,7 +61,7 @@ export const learnColumn: LearnColumnType[] = [
         defaultMessage='Weekly rewards for forecasting the market. For advanced users.'
       />
     ),
-    icon: <IconMarketUp color={SemanticColors.primary} size='medium' />,
+    icon: <Image name='bars' width='20px' height='24px' />,
```

Comment:
there is actually a bar chart icon in constellation too. `IconChartBar`

Reply:
The one in constellation has 3 bars vs the one they want only has 2, that's why I had to insert a custom bar image